src/database/operations/load/loadProposal.ts

```diff
@@ -0,0 +1,14 @@
+import { db } from '../../db';
+import { NotFoundError } from '../../../errors';
+import type { JsonResultSet, Proposal } from '../../../types';
+
+export const loadProposal = async (id: number): Promise<Proposal> => {
+	const result = await db.sql<JsonResultSet<Proposal>>('proposals.selectById', {
+		id,
+	});
+	const proposal = result.rows[0]?.object;
+	if (proposal === undefined) {
+		throw new NotFoundError(`The proposal was not found (id: ${id})`);
```

Comment:
In contrast to my above comment, this one makes sense to be a `NotFoundError` and can be easily reproduced (test covered it) because it's an HTTP GET and SQL SELECT.